applications/nrf_desktop/configuration/nrf52840dongle_nrf52840/prj_shell.conf

```diff
@@ -62,7 +62,6 @@ CONFIG_GPIO=y
 
 CONFIG_SERIAL=n
 CONFIG_UART_INTERRUPT_DRIVEN=n
-CONFIG_CONSOLE=n
```

Comment:
Maybe we could consider disabling `CONFIG_CONSOLE`, `CONFIG_UART_CONSOLE` and `CONFIG_SERIAL` for nRF Desktop application by default? It's often enabled by default by boards in Zephyr and we actually do not use it.

Reply:
We could also globally disable `CONFIG_USE_SEGGER_RTT` if we do not use logs nor shell.

Reply:
it is not possible to do it using Kconfig default as these configurations: CONFIG_CONSOLE, CONFIG_UART_CONSOLE and CONFIG_SERIAL are explicitly selected with y syntax: For example:

``CONFIG_CONSOLE=y``

Reply:
You are right, the mentioned syntax is used for quite a lot of boards. Let's leave it then